test/unit/proj_errno_string_test.cpp

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ *
+ * Project:  PROJ
+ * Purpose:  Unit test for proj_errno_string()
+ * Author:   Kurt Schwehr <schwehr@google.com>
```

Comment:
Please change the author.

Reply:
Don't want credit for my crappy test? :)